Core/src/org/sleuthkit/autopsy/datamodel/Accounts.java

```diff
@@ -77,11 +78,16 @@
 public class Accounts extends Observable implements AutopsyVisitableItem {
 
     private static final Logger LOGGER = Logger.getLogger(Accounts.class.getName());
-    private static final BlackboardArtifact.Type CREDIT_CARD_ACCOUNT_TYPE = new BlackboardArtifact.Type(TSK_CREDIT_CARD_ACCOUNT);
+    private static final BlackboardArtifact.Type ACCOUNT_TYPE = new BlackboardArtifact.Type(BlackboardArtifact.ARTIFACT_TYPE.TSK_ACCOUNT);
     @NbBundle.Messages("AccountsRootNode.name=Accounts")
     final public static String NAME = Bundle.AccountsRootNode_name();
 
     /**
+     * This is a secret handshake with
+     * org.sleuthkit.autopsy.keywordsearch.TermComponentQuery
+     */
+    private static final String CREDIT_CARD_NUMBER_ACCOUNT_TYPE = "Credit Card Number";
```

Comment:
I had been using Credit Card Number as the main thing we are really looking for and finding is the account numbers, but I think it is fine to use just "Credit Card".  I will review the variable names too.